spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
If we take the GCP-centered view, which is the current structure, then `spring-cloud-gcp-integration` doesn't make sense because all of the spring integrations would go into the respective GCP service modules like `spring-cloud-gcp-pubsub` or `spring-cloud-gcp-bigquery`. We can also take the corss-product view, which will result in modules like `spring-cloud-gcp-pubsub-integration` and `spring-cloud-gcp-biquery-integration` -- this might be too much.